### ISSUE TITLE
cr_pdb_process has unused arguments

### ISSUE THREAD

**Issue by skypjack** [fungos/cr#53]
The function in the object is defined as:
```
bool static cr_pdb_process(const std::string &source,
                           const std::string &desination) {
    std::string folder, fname, ext, orig_pdb;
    cr_split_path(desination, folder, fname, ext);
    bool result = cr_pdb_replace(desination, fname + ".pdb", orig_pdb);
    result &= cr_copy(orig_pdb, cr_replace_extension(desination, ".pdb"));
    return result;
}
```
As you can see, `source` isn't used at all. This triggers **a lot** of warnings when including `cr`.
Is it possible to suppress them by removing the unused argument from the function definition?
I can also open a PR for that, just let me know if it makes sense to you to update this function.

Thanks.

**Comment by skypjack** [comment]
`cr_dl_header_handler` and `cr_dl_header_handler` suffer from the same problem, they don't use parameter `size` afaik.

**Comment by fungos** [comment]
Sure, the PR is welcome :)

**Comment by fungos** [comment]
Merged #54
